content/storage/sql-server/automate-backup-database-sql-server-in-docker-container.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Automate Backup database SQL Server in docker container
+showMetadata: true
+editable: true
+showToc: true
+---
+
+Hi, everyone. This is first my post in dev.to and my first time for write article in second language. today I want to share some trick for backup your database from docker container. let's begin...
+
+**Note:** I assume your already running container. 
+### 1. Write T-SQL for backup your database.
+For example I use TutorialDB database for demonstrate purpose. Your can create same database click [Create a database](https://docs.microsoft.com/en-us/sql/azure-data-studio/quickstart-sql-server?view=sql-server-ver15#create-a-database)
+```
+PRINT "Start Backup process...";
+DECLARE @MyFileName varchar(200)
+SELECT @MyFileName=N'/tmp/Backup_' + convert(nvarchar(20),GetDate(),112)+'_'+convert(nvarchar(20),GetDate(),108)+ '.bak'
+BACKUP DATABASE [TutorialDB] TO DISK=@MyFileName
+PRINT "Finished backup process...";
+```
+Then save above T-SQL to backup-database.sql
+### 2. Copy backup-database.sql file to your container.
+For example I just copy it to tmp directory. You can copy it to any your directory.
+```
+docker cp ./backup-database.sql example-database:/tmp
+```
+<!-- ![image](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/nwiuc0jggck83goacj1u.png) -->
+Then go inside container by cli as below. <br />
+```
+docker exec -it example-database /bin/bash
+$ ls -lh /tmp
+```
+<br />
+Then see in tmp directory as below.
+<br />
+
+![image](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/tw5e3z8anxenfrs62tzw.png)
+### 3. Update packages list to new version.
+Go inside your container by docker exec cli as below. <br />
```

Comment:
Same as above empty line before a new topic and remove br with adding new empty line below.